examples/minimum-required.ts

```diff
@@ -1,15 +1,15 @@
 // use package in production
 // const uc = require("uc-integration-api");
-import uc from "../index";
-import MediaPlayer, { ATTRIBUTES, STATES } from "../lib/entities/media_player";
+import uc from "../index.js";
+import MediaPlayer, { ATTRIBUTES, STATES } from "../lib/entities/media_player.js";
 import {
   FEATURES as MEDIAPLAYER_FEATURES,
   ATTRIBUTES as MEDIAPLAYER_ATTRIBUTES,
   STATES as MEDIAPLAYER_STATES
-} from "../lib/entities/media_player";
+} from "../lib/entities/media_player.js";
 import { STATUS_CODES } from "http";
-import { DEVICE_STATES, EVENTS as API_EVENTS } from "../lib/api_definitions";
-import { CommandHandler } from "../lib/entities/entity";
+import { DEVICE_STATES, EVENTS as API_EVENTS } from "../lib/api_definitions.js";
+import { CommandHandler } from "../lib/entities/entity.js";
```

Comment:
Could this be simplified / reverted back with a single import statement?
This applies to the other examples as well.

This minimal example is meant to show what needs to be done in a separate client project. See the first two comment lines:

> // use package in production
// const uc = require("uc-integration-api");

Since this example is in the same project as the package, a relative import is used. In a client project, where this package is imported, this can easily be switched with the commented out line.